cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxMusic.java

```diff
@@ -280,7 +280,7 @@ private MediaPlayer createMediaPlayer(final String path) {
                 mediaPlayer.setDataSource(fis.getFD());
                 fis.close();
             } else {
-                if (Cocos2dxHelper.getObbFile() != null) {
+                if (Cocos2dxHelper.getObbFile() != null && Cocos2dxHelper.getObbFile().getAssetFileDescriptor(path) != null) {
```

Comment:
I think it is better to check `assetFileDescriptor ` inside the` if` sentence.

Reply:
My purpose is that checking whether the OBB file exists and whether the `assetFileDescriptor` is inside the OBB file. If the `assetFileDescriptor` is not inside the OBB file, we should continue to find it   through `this.mContext.getAssets()`.